PostHog/PostHogSDK.swift

```diff
@@ -365,25 +372,14 @@ let maxRetryDelay = 30.0
         flagCallReportedLock.withLock {
             flagCallReported.removeAll()
         }
-        PostHogSessionManager.shared.endSession {
-            self.resetViews()
-        }
-        PostHogSessionManager.shared.startSession()
+        PostHogSessionManager.shared.resetSession()
 
         // reload flags as anon user
         if shouldReloadFlagsForTesting {
             reloadFeatureFlags()
         }
     }
 
-    private func resetViews() {
-        #if os(iOS)
-            if config.sessionReplay, featureFlags?.isSessionReplayFlagActive() ?? false {
-                replayIntegration?.resetViews()
```

Comment:
how the views are reset if we don't call `replayIntegration?.resetViews()` anymore?